src/components/Dashboard/CompletedIa.vue

```diff
@@ -18,14 +18,16 @@
 
 <script lang="ts">
 import { mapState } from 'vuex'
-import { Component, Vue } from 'vue-property-decorator'
+import { Component, Mixins } from 'vue-property-decorator'
+import { CommonMixin } from '@/mixins'
 
 @Component({
+  mixins: [CommonMixin],
```

Comment:
Is this required when we are extending the class with the mixin as well?

Reply:
It's apparently not required since we have many TS components that don't have this and still work.
In theory this better decorates the component/mixin, but I don't really know for sure what this does.
Since I was in this area of code, I added it, FWIW.